site/app/templates/forum/ForumBar.twig

```diff
@@ -158,6 +158,27 @@
 		</div>
 	{% endif %}
 </div>
+<div class="row mt-2">
+	<div class="col-3">
+		<div class="dropright">
+			<a title="Sorting options" id="dropdownLabel" role="button" data-toggle="dropdown" class="btn btn-default btn-block dropdown-toggle" data-target="#" href="#">
```

Comment:
The id `"dropdownLabel"` is already used elsewhere on the page in the "more" dropdown button. Change name of the id to avoid duplicates.